ui/apps/platform/src/Containers/Violations/Details/Deployment/ContainerConfigurationDescriptionList.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement } from 'react';
+import { DescriptionList, Flex, FlexItem } from '@patternfly/react-core';
+
+import DescriptionListItem from 'Components/DescriptionListItem';
+import { Container } from 'types/deployment.proto';
+
+import ContainerImage from './ContainerImage';
+import ContainerResourcesDescriptionList from './ContainerResourcesDescriptionList';
+import ContainerSecretDescriptionList from './ContainerSecretDescriptionList';
+import ContainerVolumeDescriptionList from './ContainerVolumeDescriptionList';
+
+function MultilineDescription({ descArr }) {
+    return (
+        <Flex direction={{ default: 'column' }}>
+            {descArr.map((desc, idx) => (
+                // eslint-disable-next-line react/no-array-index-key
+                <div key={idx}>
+                    <FlexItem>{desc}</FlexItem>
+                </div>
+            ))}
+        </Flex>
+    );
+}
+
+type ContainerConfigurationDescriptionListProps = {
+    container: Container;
+};
+
+function ContainerConfigurationDescriptionList({
+    container,
+}: ContainerConfigurationDescriptionListProps): ReactElement {
+    const { resources, volumes, secrets, config, image } = container;
+    const { command, args } = config || {};
+    return (
+        <DescriptionList isCompact isHorizontal>
+            <ContainerImage image={image} />
+            {(command?.length > 0 || args?.length > 0) && (
```

Comment:
Nit: Is this check still necessary since it is repeated for both `command` and `args` in the fragment?

Reply:
Hmm I just changed the file name for this so I didn't look too deep into what it was already doing. Let me double check soon

Reply:
Yea, you're right. Seems a bit unnecessary. Maybe we can just refactor this a bit now too